### ISSUE TITLE
Audit site for color contrast issues

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#589]
See Accessibility & Inclusion Toolkit section on [color & contrast](https://www2.gov.bc.ca/gov/content/home/accessible-government/toolkit/accessible-digital-content/colour-contrast)

#### Definition of Done
- [x] On a current deployment, visit all site pages and scan for any color contrast issues that violate web accessibility standards

**Comment by singhinderdeep** [comment]
Manually audited all the pages for any color contrast issues. Looks good.
